cli/agent_test.go

```diff
@@ -9,6 +9,11 @@ import (
 	"strings"
 	"testing"
 
+	"cdr.dev/slog"
+	"cdr.dev/slog/sloggers/slogtest"
+
+	"github.com/coder/coder/v2/testutil"
```

Comment:
Import order is messed up here. Usually due to editor inserting on autocomplete but in the wrong place and formatter not being strict to repair it :/.